homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -300,7 +300,7 @@ async def _update_sources(self) -> None:
         if self.hass.is_running:
             self.async_write_ha_state()
 
-    async def _update_playback_metadata(self, data: PlaybackContentMetadata) -> None:
+    def _update_playback_metadata(self, data: PlaybackContentMetadata) -> None:
```

Comment:
You need to decorate the function with `@callback` to ensure they are executed inside the event loop.

```python
from homeassistant.core import callback
```